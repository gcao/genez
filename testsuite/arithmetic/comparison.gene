#!/usr/bin/env gene run

# Test: Comparison operations
(print "10 < 20 = " (< 10 20))
(print "20 < 10 = " (< 20 10))
(print "10 > 5 = " (> 10 5))
(print "5 > 10 = " (> 5 10))
(print "10 == 10 = " (== 10 10))
(print "10 == 20 = " (== 10 20))