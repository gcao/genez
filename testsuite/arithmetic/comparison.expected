10 < 20 =  true
20 < 10 =  false